types/create-react-class/create-react-class-tests.ts

```diff
@@ -75,32 +75,20 @@ const ClassicComponent: React.ClassicComponentClass<Props> = createReactClass<Pr
     },
 });
 
-const ClassicComponentNoProps: React.ClassicComponentClass = createReactClass({
+const ClassicComponentNoProps: createReactClass.ClassicComponentClass = createReactClass({
     render() {
         return DOM.div();
     },
 });
 
-const ClassicComponentNoState: React.ClassicComponentClass<{ text: string }> = createReactClass<{ text: string }>({
+const ClassicComponentNoState: createReactClass.ClassicComponentClass<{ text: string }> = createReactClass<
+    { text: string }
+>({
     render() {
         return DOM.div(this.props.text);
     },
 });
 
-// React.createFactory
```

Comment:
I've moved all these tests to `react` and `react-dom`. They were actually testing that non-standard instance methods were preserved in these APIs which is unrelated to `create-react-class`.